scripts/targetctl

```diff
@@ -56,7 +67,41 @@ def restore(from_file):
 def clear(unused):
     RTSRoot().clear_existing(confirm=True)
 
-funcs = dict(save=save, restore=restore, clear=clear)
+def suspend(unused):
+    if not os.path.isdir(os.path.dirname(svc_suspended_runfile)):
+        os.mkdir(os.path.dirname(svc_suspended_runfile))
+
+    if os.path.exists(svc_suspended_runfile):
+        print('Service already suspended.')
+    else:
+        # Create run-file that indicates that the service is suspended.
+        # Note that we do this before clearing the state. If the operation
+        # fails mid-way then we will be prevented from making modifications
+        # to a partially destroyed state.
+        with open(svc_suspended_runfile, 'a'):
+            pass
+
+        clear(None)
+        print('Service suspended.')
+
+def resume(config_file):
+    if os.path.exists(svc_suspended_runfile):
+        restore(config_file, check_suspended=False)
+        os.remove(svc_suspended_runfile)
+        print('Service resumed.')
+    else:
+        print('Service was not suspended.')
+
+def factory_reset(config_file):
+    clear(None)
+    if os.path.exists(svc_suspended_runfile):
+        os.remove(svc_suspended_runfile)
+    save(config_file)
```

Comment:
What does this accomplish?

Reply:
`clear` destroys the current state.
`save` overwrites the config file with the empty state.